net/netifyd/Makefile

```diff
@@ -0,0 +1,113 @@
+# Netify Agent
+# Copyright (C) 2016-2018 eGloo, Incorporated
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+# Name, version and release number
+# The name and version of your package are used to define the variable to point to the build directory of your package: $(PKG_BUILD_DIR)
+PKG_NAME:=netifyd
+PKG_VERSION:=2.67
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Darryl Sokoloski <darryl@egloo.ca>
+PKG_LICENSE:=GPL-3.0+
+
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+PKG_CONFIG_DEPENDS:=CONFIG_LIBCURL_ZLIB
+
+# Source settings
+
+# Disable the following and enable SOURCE_DIR below for local test builds.
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.com/netify.ai/public/netify-agent.git
+PKG_SOURCE_DATE:=2018-07-03
+PKG_SOURCE_VERSION:=b47cd92b6d9ad5d377c19ed3457490eef4e8a970
+PKG_MIRROR_HASH:=aff279e417d4edb755ab2560e6672bb1e9694340f2efc3dafa8da35244ae6018
+
+# Local source builds (create symlink to source here)
+#SOURCE_DIR:=$(TOPDIR)/netify-daemon
+
+#include $(INCLUDE_DIR)/uclibc++.mk
+include $(INCLUDE_DIR)/package.mk
+
+# Package definition
+define Package/netifyd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Netify Agent
+  URL:=http://www.netify.ai/
+  DEPENDS:=+libcurl +libmnl +libnetfilter-conntrack +libjson-c +libpcap +zlib +libpthread +uclibcxx +libstdcpp
```

Comment:
Instead of adding both uclibcxx and libstdcpp as deps you probably want to use $(CXX_DEPENDS) if it works with both uclibcxx and libstdcpp that is. 

This should be a pretty good template
https://github.com/openwrt/packages/blob/master/utils/smartmontools/Makefile

Reply:
Unfortunately by doing this, I loose support for <unordered_map> which is essential for performance reasons.  I require libstdcpp for this...  is that acceptable, or is there another way to deal with that?

Reply:
In that case, drop uclibcxx?